hoomd/__init__.py

```diff
@@ -90,3 +92,14 @@ def _hoomd_sys_excepthook(type, value, traceback):
 sys.excepthook = _hoomd_sys_excepthook
 
 __version__ = "2.9.0"
+
+__git_sha1__ = _hoomd.__git_sha1__
+
+__git_refspec__ = _hoomd.__git_refspec__
+
+__compile_flags__ = _hoomd.hoomd_compile_flags()
+
+__cuda_version__ = _hoomd.__cuda_version__
```

Comment:
`get_cuda_version_tuple()` feeds into `__cuda__version__`. Should we change it to return a string like other `__version__` attributes?